### PR TITLE
Add support for const object members

### PR DESCRIPTION
**Pull request by bioball**
If `local` is missing:

Screenshot 2024-11-05 at 1 36 25 PM

If used in 0.26 and earlier:

Screenshot 2024-11-05 at 1 35 09 PM

### CODE REVIEW

**Review by stackoverflow — APPROVED** [review]
Looks good. Just needs a `spotlessApply`.

File: src/main/kotlin/org/pkl/intellij/annotator/PklMemberAnnotator.kt [hunk 1]
```diff
@@ -413,6 +416,25 @@ class PklMemberAnnotator : PklAnnotator() {
       }
     }
 
+    if (module.effectivePklVersion >= PklVersion.VERSION_0_27) {
+      if (constModifier != null && owner is PklObjectMember) {
+        if (localModifier == null) {
```
Comment (holzensp) [inline src/main/kotlin/org/pkl/intellij/annotator/PklMemberAnnotator.kt:421]: `&&` into a single `if`?

File: src/main/kotlin/org/pkl/intellij/util/Feature.kt [hunk 2]
```diff
@@ -17,17 +17,28 @@ package org.pkl.intellij.util
...
-  val requiredVersion: PklVersion,
+sealed interface Feature {
+  val featureName: String
+  val requiredVersion: PklVersion
   val predicate: (PsiElement) -> Boolean
-) {
+  val message: String?
+    get() = null
 
   fun isSupported(module: PklModule): Boolean = module.effectivePklVersion >= requiredVersion
```
Comment (holzensp) [inline src/main/kotlin/org/pkl/intellij/util/Feature.kt:29]:
```suggestion
  fun isSupported(module: PklModule): Boolean = module.effectivePklVersion >= requiredVersion

  fun featureMessage(): String = message ?: "Unsupported feature: $featureName"
```

^^ Seems reusable enough

**Review by holzensp — APPROVED** [review]
LGTM; one minor re-use suggestion

**Pull request merged**